rskj-core/src/main/java/co/rsk/remasc/Remasc.java

```diff
@@ -19,6 +19,8 @@
 package co.rsk.remasc;
 
 import co.rsk.config.RemascConfig;
+import co.rsk.core.bc.SelectionRule;
+import com.sun.org.apache.bcel.internal.generic.Select;
```

Comment:
What is this import?

You can enable automatic imports optimization if you're using IntelliJ, or manually delete unused imports if you prefer.